### PR TITLE
Added css linter, scripts and eslint react linter

### PR DESCRIPTION
**Pull request by magsout**
@miketaylr  ?

### CODE REVIEW

**Comment by magsout** [comment]
Hum I need to update my node and npm version, because package-lock.json is not updated

**Comment by miketaylr** [comment]
> Hum I need to update my node and npm version, because package-lock.json is not updated

Don't worry about that I filed https://github.com/webcompat/webcompat-metrics/issues/17 to get rid of it.

File: .eslintrc [hunk 1]
```diff
@@ -5,10 +5,16 @@
   },
   "extends": [
     "eslint:recommended",
-    "plugin:prettier/recommended"
+    "prettier",
+    "prettier/react",
+    "eslint-config-i-am-meticulous/react",
```
Comment (miketaylr) [inline .eslintrc]: > an ESLint shareable config for meticulous people.

haha cool.

Comment (magsout) [reply .eslintrc]: heh, we are meticulous 💃

**Pull request merged**